### PR TITLE
Adding Dutch translation

### PR DESCRIPTION
**Pull request by ruditimmermans**
I have tranlate the string also into Dutch language...

### CODE REVIEW

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  <string name="time">Tijd</string>
+  <string name="dosage">Dosering</string>
+  <string name="yes">Ja</string>
+  <string name="ok">OK</string>
+  <string name="cancel">Annuleren</string>
+  <string name="confirm">Bevestigen</string>
+  <string name="are_you_sure_delete_medicine">Weet u zeker dat u het geneesmiddel wilt verwijderen?</string>
+  <string name="no_reminders">Geen herinneringen ingesteld</string>
+  <plurals name="som_herinneringen">
+       <item quantity="een">%1$d herinnering\n%2$s</item>
```
Comment (Futsch1) [inline app/src/main/res/values-nl/strings.xml]: The attribute must not be translated - this should stay "one" and "other" in line 22

**Comment by Futsch1** [comment]
Thanks a lot for the translation and your PR! There is one minor issue, as soon as you fix it I will merge and include in the next release.

File: app/src/main/res/values-nl/strings.xml (hunk 1 above)
Comment (ruditimmermans) [reply app/src/main/res/values-nl/strings.xml:21]: So you mean line 21 and 22 should not be translated ?

Comment (ruditimmermans) [reply app/src/main/res/values-nl/strings.xml]: Ok sorry it should be fixed now ;)

File: app/src/main/res/values-nl/strings.xml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  <string name="delete">Verwijderen</string>
+  <string name="time">Tijd</string>
+  <string name="dosage">Dosering</string>
+  <string name="yes">Ja</string>
+  <string name="ok">OK</string>
+  <string name="cancel">Annuleren</string>
+  <string name="confirm">Bevestigen</string>
+  <string name="are_you_sure_delete_medicine">Weet u zeker dat u het geneesmiddel wilt verwijderen?</string>
+  <string name="no_reminders">Geen herinneringen ingesteld</string>
+  <plurals name="som_herinneringen">
```
Comment (Futsch1) [inline app/src/main/res/values-nl/strings.xml]:
```suggestion
  <plurals name="sum_reminders">
```

**Pull request merged**